### ISSUE TITLE
PostBuild.proj exit code 2 or 3

### ISSUE THREAD

**Issue by geoffrey-vl** [mrward/monodevelop-dnx-addin#1]
Building the solution exits with exit-code 2 on Linux, exit-code 3 on Windows. The issue can be solved by downloading the ILRepack Nuget package, and next edit its executable path in PostBuild.proj to match the Nuget downloaded package, for example:

<ILRepackExePath Condition=" '$(ILRepackPath)' == '' ">$(SolutionDir)packages\ILRepack.2.0.8\tools\ILRepack.exe</ILRepackExePath>

**Comment by mrward** [comment]
Instead of doing that you can run `NuGet.exe restore MonoDevelop.Dnx.sln` from the directory containing the solution.

I should add that to the readme or maybe look at using a before.sln targets file to automatically restore on the build. Or it may just be simpler to add ILRepack to the GitHub repository instead of using NuGet.

**Comment by geoffrey-vl** [comment]
Indeed, that way none of the current code could brake because of changes in ILRepack.

Do you plan to host the addin in a repo?

**Comment by mrward** [comment]
I was planning on publishing the addin at some point to the [MonoDevelop addin-repo](http://addins.monodevelop.com/) and to my own [addin repo](https://github.com/mrward/monodevelop-addins). Probably just after Microsoft release ASP.NET 5 RC 1, which should happen sometime this month.
